AutomaticPackages.md

```diff
@@ -60,10 +60,12 @@ This tool makes use of [Ketarin](https://chocolatey.org/packages/ketarin). Ketar
 1. Check the config in `$env:ChocolateyInstall\lib\ChocolateyPackageUpdater\tools\chocopkgup\chocopkgup.exe.config`. The `PackagesFolder` key should point to where your repository is located.
 1. Create a scheduled task (in Windows). This is the command (edit the path to `cmd.exe` accordingly): `C:\Windows\System32\cmd.exe /c c:\tools\chocolateypackageupdater\ketarinupdate.cmd`
 1. Alternatively to stop the command window from opening on Windows, you can create a VBS script as well and put the path to the `.vbs` file instead of `ketarinupdate.cmd` as the command to run. The file should have the following:
-~~~vb
-Set objShell = WScript.CreateObject("WScript.Shell")
-objShell.Run("C:\tools\ChocolateyPackageUpdater\ketarinupdate.cmd"), 0, True
-~~~
+
+    ```vb
```

Comment:
Sorry - I switched it to match the same style as the other code sample in this file. 😄 I've changed both to be `~~~` now, if that's the convention.